CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.2.0 (unreleased)
+
+NOTES:
+
+* This Go module has been updated to Go 1.18 per the Go support policy. Any consumers building on earlier Go versions may experience errors ([#171](https://github.com/hashicorp/terraform-provider-http/issues/171)).
```

Comment:
Similar note as random about either dropping this changelog for practitioners (unless there are specific net/http changes we want to mention) or tailoring it for provider developers